lib/web/css/source/lib/_resets.less

```diff
@@ -51,14 +51,17 @@
         border: 0;
```

Comment:
please move img styles to this block, as they were overridden
```
      max-width: 100%;
      height: auto;
```

Reply:
Done, Yes it better to have the img styles merged under 1 selector.